packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/queue/queue.ts

```diff
@@ -0,0 +1,389 @@
+import * as path from 'path';
+import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as events from '@aws-cdk/aws-events';
+import * as events_targets from '@aws-cdk/aws-events-targets';
```

Comment:
is it common to import with underscores?

Reply:
Yes it is. Also, using hyphen doesn't work 🙈